File: PAP2223/src/tp01/Jeu.java
package tp01;

import java.util.Random;
import java.util.Scanner;

public class Jeu {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		
		// create instance of Random class
		Random rand = new Random();
		int secret = rand.nextInt(100) + 1;
	
		boolean trouve = false;
		do {
			System.out.println("Entrez un nombre entre 1 et 100");
			int nbr = scan.nextInt();

			if (nbr < secret)
				System.out.println("Trop petit");
			else if (nbr > secret)
				System.out.println("Trop grand");
			else {
				System.out.println("Super tu l'as trouvé");
				trouve = true;
			}
		} while (!trouve);
		System.out.println("Fin");
		scan.close();
	}

}

File: PAP2223/python/jeu.py
import random
secret = random.randint(0,100)
#print(secret)
n=102 #pour rentrer dans la boucle
while n != secret:
    n= int(input("Entrez un nombre \n"))
    if n<secret :
        print("Le nombre est trop petit")
    elif n>secret :
        print("Le nombre est trop grand")
print("Super tu as trouvé !!!")
        